<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PIRPanelSelectionForm</class>
 <widget class="QWidget" name="PIRPanelSelectionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Panels</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>8</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>782</width>
        <height>462</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="mainCheckBox">
         <property name="text">
          <string>Main Panel - power, volume, and channel controls</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="utilityCheckBox">
         <property name="text">
          <string>Utility Panel - commonly used controls</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="keypadCheckBox">
         <property name="text">
          <string>Keypad Panel - numeric value entry</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="menuCheckBox">
         <property name="text">
          <string>Menu Panel - enter, exit, and navigate menus</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="mediaCheckBox">
         <property name="text">
          <string>Media Panel - control over recorded data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="media2CheckBox">
         <property name="text">
          <string>Media2 Panel - additional media controls</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="recordCheckBox">
         <property name="text">
          <string>Program/Record Panel - control over memory and storage</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="tvCheckBox">
         <property name="text">
          <string>TV Panel - Teletext and Picture-in-Picture</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="inputCheckBox">
         <property name="text">
          <string>Input Panel - manage data sources</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="adjustCheckBox">
         <property name="text">
          <string>Adjust Panel - modify audio and video</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="acCheckBox">
         <property name="text">
          <string>A/C Panel - air conditioner controls</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="favoritesCheckBox">
         <property name="text">
          <string>Favorites Panel - memorized keysets</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
